mantidproject/mantid
# Reflectometry GUI restructuring 6: file tidying

**Description of work.**
Ongoing work to integrate the `JobTreeView` into the `ISIS Reflectometry` GUI. This PR is being merged to a feature branch because not all functionality is implemented yet.

This PR tidies files into subdirectories and removes unused files. It therefore touches a lot of files, but the changes are simple.

- The Batch and MainWindow components have been renamed in line with the simplified names for other components and they have been moved to their own subdirectories under `GUI`
- Common components have been moved to the `Common` subdirectory
- The remaining components have been moved to the relevant subdirectory for the GUI component that uses it.

**Report to:** nobody

**To test:**
Check that the `ISIS Reflectometry` GUI opens and that all inputs function, although note that actual processing of runs is still disabled. In particular:
- Enter new rows/groups in the table and edit cells.
- Add/remove Batch tabs on the left tab bar using the Batch menu and X button to close.
- Search and transfer runs. Note you first need to log in to the catalog by the main mantidplot `Catalog` menu, then enter e.g. instrument `INTER` and experiment ID `1120015` and click `Search`. The runs should appear in the list below. Select some and click `Transfer` and they should appear in the main table.
- Edit values in the Event, Experiment or Instrument tab.
- When you click `Process` or `Autoprocess`, although actual processing won't happen, some GUI inputs will be disabled as if it were (i.e. anything that would affect the reduction is disabled during processing). Clicking `Pause` should re-enable them.

**To follow**

*There is no associated issue.*

*This does not require release notes* because it only contains under-the-hood changes.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
